docs/architecture/porting-existing-aspnet-apps/deployment-strategies.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Deployment strategies
+description: What deployment strategies can teams use while migrating from ASP.NET to .NET Core? Can an incremental migration allow side-by-side deployment of .NET Framework and .NET Core apps, providing a seamless end user experience?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Deployment strategies
+
+One consideration as you plan the migration of your large ASP.NET app to ASP.NET Core is how you will deploy the new app. With ASP.NET, deployment options were limited to IIS on Windows. With .NET Core, a much wider array of deployment options are available.
+
+## Cross-platform options
+
+Because .NET Core supports Linux, you may now find some hosting options available that weren't a consideration previously. Some hosting providers may offer attractive features or pricing for Linux-based hosting that make it an attractive option, or you may have your own non-Windows servers you'd like to leverage as part of your new app deployment strategy. .NET Core opens to door to these options.
```

Comment:
```suggestion
Because .NET Core supports Linux, you may now find some hosting options available that weren't a consideration previously. Some hosting providers may offer attractive features or pricing for Linux-based hosting that make it an attractive option, or you may have your own non-Windows servers you'd like to leverage as part of your new app deployment strategy. .NET Core opens the door to these options.
```

Reply:
Would it be interesting to phrase this as "Linux-based hosting may be preferable either because your organization has infrastructure or expertise, or because hosting providers offer attractive features or pricing for Linux-based hosting"?